src/ObjectGraph.es6.js

```diff
@@ -98,18 +98,21 @@ ObjectGraph.prototype.blacklistedObjects = [
   value => value !== undefined
 );
 ObjectGraph.prototype.blacklistedProperties = [
-  ['MimeType', 'enabledPlugin'],
-    // MimeType's enabledPlugin will return a new MimeType object in Chrome
-    // (version<=42), which will cause a infinite recursion in the object graphs.
+  [ 'MimeType', 'enabledPlugin' ],
+    // MimeType's enabledPlugin will return a new MimeType object
```

Comment:
nit: Let's indent this in-line with the `[`, and put it above the inner array (`[ 'MimeType', 'enabledPlugin' ]`).